ui/components/ui/icon-button/icon-button.js

```diff
@@ -57,7 +57,7 @@ IconButton.propTypes = {
   onClick: PropTypes.func.isRequired,
   Icon: PropTypes.object.isRequired,
   disabled: PropTypes.bool,
-  label: PropTypes.string.isRequired,
+  label: PropTypes.string,
```

Comment:
Is this needed?

Reply:
Actually.. since now we are using button-icon, I think we can get rid of the changes to make label optional. Thanks!